BE/Api.Access.Layer/Controllers/UserController.cs

```diff
@@ -32,6 +32,15 @@ public async Task<IActionResult> Delete([FromBody] string email)
 
         }
 
+        [ProducesResponseType((int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
+        [HttpGet("read")]
+        public async Task<IActionResult> Read([FromBody] string email)
```

Comment:
Please rename this method to 'GetUser'. Also, it is a good practice to name our methods in such a way that when you read the title you know what that method does (without the need to look at its implementation of it).

So in this case, as you get user by email parameter the name of the method should be like "GetUserByEmail".

Reply:
Okay, my fault, I will fix it. Thanks for suggestion about giving name of methods I will try to improve it.